src/wagtail_personalisation/models.py

```diff
@@ -302,3 +302,16 @@ def personalisation_metadata(self):
             metadata = PersonalisablePageMetadata.objects.create(
                 canonical_page=self, variant=self)
         return metadata
+
+    def get_sitemap_urls(self, request=None):
+        # Do not generate sitemap entries for variants.
+        if not self.personalisation_metadata.is_canonical:
+            return []
+        try:
```

Comment:
I prefer to [check version explicitly](https://github.com/wagtail/wagtail/blob/v2.2/wagtail/__init__.py).

It will be easier to find places with version checks when we decide to drop support for versions below 2.2 (or any other version in other instances).

Reply:
I've changed the way the version checks work.